package io.github.devDias.domain.exception;

public class PedidoNaoEncontradoException extends RuntimeException {

    public PedidoNaoEncontradoException() {
        super("Pedido n√£o encontrado.");
    }
}
